.github/workflows/tests.yml

```diff
@@ -0,0 +1,41 @@
+name: Tests
```

Comment:
Maybe do this in a separate commit (if not too much work), so it's easier to find back later?